### PR TITLE
Update rocket hash

### PR DESCRIPTION
**Pull request by hmtanbir**
Solved feeback.
Previous: #2212

### CODE REVIEW

**Comment by hmtanbir** [comment]
@pirj Would you please check this PR ?

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ Style/EachWithObject:
 
 # ruby 1.8.7 only had support for hash rockets
 Style/HashSyntax:
-  EnforcedStyle: hash_rockets # default: ruby19
+  EnforcedStyle: ruby19 # default: ruby19
```
Comment (pirj) [inline .rubocop_todo.yml]: You may freely remove this setting since it's the default.

File: Changelog.md [hunk 2]
```diff
@@ -344,10 +344,10 @@ Enhancements:
 * Improve controller and routing spec calls to `routes` by using `yield`
   instead of `call`. (Anton Davydov, #1308)
 * Add support for `ActiveJob` specs as standard `RSpec::Rails::RailsExampleGoup`s
-  via both `:type => :job` and inferring type from spec directory `spec/jobs`.
+  via both `type: :job` and inferring type from spec directory `spec/jobs`.
```
Comment (pirj) [inline Changelog.md]: Please rollback Changelog changes, it's unnecessary.

File: features/step_definitions/additional_cli_steps.rb [hunk 3]
```diff
@@ -1,10 +1,10 @@
 begin
   require "active_job"
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError # rubocop:disable Lint/SuppressedException
 end
 begin
   require "action_cable"
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError # rubocop:disable Lint/SuppressedException
```
Comment (pirj) [inline features/step_definitions/additional_cli_steps.rb]: Thanks, sharp eye 👍

File: lib/rspec/rails/configuration.rb [hunk 4]
```diff
@@ -25,33 +25,33 @@ class Configuration
...
+      helper: %w[spec helpers],
+      job: %w[spec jobs],
+      mailer: %w[spec mailers],
+      model: %w[spec models],
+      request: %w[spec (requests|integration|api)],
+      routing: %w[spec routing],
+      view: %w[spec views],
+      feature: %w[spec features],
+      system: %w[spec system],
+      mailbox: %w[spec mailboxes]
```
Comment (pirj) [inline lib/rspec/rails/configuration.rb]: Maybe keep the alignement?

**Comment by pirj** [comment]
@hmtanbir Please squash commits. Thanks for the effort!

File: Changelog.md (hunk 2 above)
Comment (JonRowe) [reply Changelog.md]: This still needs to happen.

Comment (JonRowe) [reply Changelog.md]: Decided it was ok

**Comment by pirj** [comment]
👍 
Thanks!

**Pull request merged**